.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - Enhancement
+  - Hacktoberfest
+  - Bug
+  - ⚡ To review
```

Comment:
* Should we also add `😬 Reviewed`?
    * Why: If something has not been merged but it's reviewed, it's our responsibility to merge it, not the end user one. Doesn't it?